File: Python/25.py
printf=print
printf("Esse programa calcula a media de N alunos e determina quantos passaram e quantos reprovaram!\n\n")
n = int(input("\n\nQuantidade de alunos: "))
rep=0
ap=0
media=0
i=1
while(i<=n):
	nota = float(input(f"\n\nDigite a nota do aluno {i}: "))
	if(nota>=7):
		ap=ap+1
	else:
		rep=rep+1

	media=media+nota
	i=i+1

printf (f"\n\n{ap} alunos APROVADOS!")
printf (f"\n\n{rep} alunos REPROVADOS!")
media=media/n
printf (f"\n\nA media da turma é {media}!")
printf("\n\nObrigado por usar nosso sistema!\n\n")

File: C/25.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float i,n,rep,ap,media,nota;
	printf("Esse programa calcula a media de N alunos e determina quantos passaram e quantos reprovaram!\n\n");
	printf ("\n\nQuantidade de alunos: ");
	scanf ("%f", &n);

	i=1;
	while(i<=n){
		printf ("\n\nDigite a nota do aluno %.0f: ",i);
		scanf ("%f", &nota);
		if(nota>=7){
			ap++;
		}else{
			rep++;
		}
		media=media+nota;
		i++;
	}
	printf ("\n\n%.0f alunos APROVADOS!", ap);
	printf ("\n\n%.0f alunos REPROVADOS!", rep); 	
	printf ("\n\nA media da turma é %.2f!", media/n);
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}

